sim/core/rage.go

```diff
@@ -1,11 +1,15 @@
 package core
 
-import ()
-
 const MaxRage = 100.0
 
+const RageFactor = 3.75 / 274.7
+
 var RageBarAuraID = NewAuraID()
 
+var HitFactor float64 = 0
+
+var BaseSwingSpeed float64 = 0
```

Comment:
You can create these variables in the local function scope, instead of making them global. In Go that looks like this:

var hitFactor float64
var baseSwingSpeed float64

if hitEffect.IsMH() {
  hitFactor = ...
  baseSwingSpeed = ...
}